client/windows/CAdvmapInterface.h

```diff
@@ -56,6 +56,16 @@ class CTerrainRect
 	SDL_Surface * fadeSurface;
 	EMapAnimRedrawStatus lastRedrawStatus;
 	CFadeAnimation * fadeAnim;
+
+	int3 swipeInitialMapPos;
+	int3 swipeInitialRealPos;
+	bool isSwiping;
+	static constexpr float SwipeTouchSlop = 16.0f;
+
+	void handleHover(const SDL_MouseMotionEvent &sEvent);
+#ifdef VCMI_ANDROID

```

Comment:
I thought about doing it for desktop later. Preferably using right or middle mouse button.